### PR TITLE
Feature/1902/remove about tutor title from profile and renamed tutor-profile staff to user-profile

### PR DESCRIPTION
**Pull request by mhevyk**
- Renamed `tutor-profile` to `user-profile`
- Removed "about tutor" title from profile (change made in file [AboutTutorBlock.tsx](https://github.com/ita-social-projects/SpaceToStudy-Client/pull/1928/files#diff-b491f5499610bcb50b099124fb2e4ca2ae1e1e8ab27f38cfb31229cefc7c2ccc)) and added following line of code:

```jsx
if (accordionItems.length === 0) {
    return null
}
```
- Fixed https://github.com/ita-social-projects/SpaceToStudy-Client/pull/1920#discussion_r1615832056

### CODE REVIEW

**Comment by OliaHrytsak** [comment]
Please, consider the possibility of fixing TS-errors in RatingBlock.tsx

File: src/containers/user-profile/profile-info/ProfileContainerDesktop.jsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ const ProfileContainerDesktop = ({
   doneItems,
   chipItems
 }) => {
+  console.log(userData.professionalSummary)
```
Comment (Mav-Ivan) [inline src/containers/user-profile/profile-info/ProfileContainerDesktop.jsx:21]: Do we need this console.log ?

File: src/containers/user-profile/about-tutor-block/AboutTutorBlock.tsx [hunk 2]
```diff
@@ -1,61 +1,69 @@
...
+  if (accordionItems.length === 0) {
+    return null
+  }
+
+  return (
+    <Box sx={styles.root}>
+      <Typography
+        sx={styles.title}
+        variant={
+          isMobile ? TypographyVariantEnum.Button : TypographyVariantEnum.H5
```
Comment (Mav-Ivan) [inline src/containers/user-profile/about-tutor-block/AboutTutorBlock.tsx:46]: It`s better to move it above

File: src/containers/user-profile/profile-info/ProfileContainerDesktop.jsx (hunk 1 above)
Comment (mhevyk) [reply src/containers/user-profile/profile-info/ProfileContainerDesktop.jsx]: I did not notice it, thanks for your thorough review. Removed redundant `console.log`

File: src/containers/user-profile/about-tutor-block/AboutTutorBlock.tsx (hunk 2 above)
Comment (mhevyk) [reply src/containers/user-profile/about-tutor-block/AboutTutorBlock.tsx]: It is useful remark, fixed

**Comment by mhevyk** [comment]
> Please, consider the possibility of fixing TS-errors in RatingBlock.tsx

Really, I did not notice, that `RatingType` type was not included in global type export and there were errors. I fixed it

**Review by Mav-Ivan — APPROVED** [review]

**Review by OliaHrytsak — APPROVED** [review]

**Review by byavdoshniak — APPROVED** [review]

File: src/pages/user-profile/UserProfile.jsx [hunk 3]
```diff
@@ -2,25 +2,25 @@ import { useCallback } from 'react'
...
 import useAxios from '~/hooks/use-axios'
-import videoImgProfile from '~/assets/img/tutor-profile-page/presentationVideoImg.png'
+import videoImgProfile from '~/assets/img/user-profile-page/presentationVideoImg.png'
 
 import { profileItems } from '~/components/profile-item/complete-profile.constants'
 import { defaultResponses } from '~/constants'
-import { responseMock } from '~/pages/tutor-profile/constants'
-import AboutTutorBlock from '~/containers/tutor-profile/about-tutor-block/AboutTutorBlock'
+import { responseMock } from '~/pages/user-profile/constants'
+import AboutTutorBlock from '~/containers/user-profile/about-tutor-block/AboutTutorBlock'
```
Comment (D1nviz) [inline src/pages/user-profile/UserProfile.jsx:18]: move components and container imports up

File: src/types/user-profile/userProfile.index.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import type { RatingType } from './types/userProfile.types'
```
Comment (D1nviz) [inline src/types/user-profile/userProfile.index.ts:1]: use alias import instead

File: src/pages/user-profile/UserProfile.jsx (hunk 3 above)
Comment (mhevyk) [reply src/pages/user-profile/UserProfile.jsx:11]: Rearranged imports

File: src/types/user-profile/userProfile.index.ts (hunk 4 above)
Comment (mhevyk) [reply src/types/user-profile/userProfile.index.ts]: Fixed

**Review by D1nviz — APPROVED** [review]

**Pull request merged**